workloads/network-perf/run_network_compare.sh

```diff
@@ -31,12 +31,8 @@ else
   compare_uuid=$2
 fi
 
-git clone https://github.com/cloud-bulldozer/touchstone
-cd touchstone
-python3 -m venv ./compare
-source ./compare/bin/activate
-pip3 install -r requirements.txt
-python3 setup.py develop
+pip3 uninstall -y touchstone
```

Comment:
is this necessary?

Reply:
it'll help us stay updated with upstream

Reply:
ack.... why not just create a python venv, and delete it at the end?

Reply:
cool